README.md

```diff
@@ -505,6 +508,17 @@ Setting `DISABLE_NETWORK_RESOURCE_PROVISIONING` to `true` will make IPAMD to dep
 
 ---
 
+#### `ENI_MANAGE_MODE` (v1.9.1+)
```

Comment:
this knob should actually control what an "Untagged" ENI should be considered as managed or unmanaged.

* if ENI have `node.k8s.amazonaws.com/no_manage`, it should always be considered as no_managed.
* if ENI have `node.k8s.amazonaws.com/manage`, it should always be considered as managed.
* if ENI is created by IPAMD(have `node.k8s.amazonaws.com/instance_id`), it should always be considered as managed.
* 
* If ENI don't fall into any category above, this knobs determines whether it's managed or unmanaged.

Reply:
yeah sure, I can add something like this.